---
layout: page
title: Visualisation of data
subtitle: Examples from prior studies
---

<br/>

<center> <b> Circos, from <a href="https://www.nature.com/articles/s41467-020-18474-w">Nature Communications, 2020</a> </b> </center>

![The image shows alignments between multiple contigs, visualised with the program Circos](https://CormacKinsella.github.io/assets/img/data_vis-treebeard-circos.png "Circos for visualising complex features and multiple alignments")

>Visualising complex features and multiple alignments. Here, showing sequence identity between virus genes and endogenous viral element regions in parasite genomes.

<br/>

<center> <b> Alphafold & Mol*, from <a href="https://www.pnas.org/doi/10.1073/pnas.2303844120">PNAS, 2023</a> </b> </center>

![The image shows protein structures, predicted by Alphafold, visualised with the program Mol*](https://CormacKinsella.github.io/assets/img/data_vis-PNAS_alphafold.png "Alphafold protein/domain structure analysis")

>Alphafold in combination with Mol* for protein domain analysis. Here, exploring predicted monomer and multimer structure of proteins involved with single-stranded DNA replication.

<br/>

<center> <b> Gene ontology enrichment, from <a href="https://www.nature.com/articles/s41467-019-13427-4">Nature Communications, 2019</a> </b> </center>

![The image shows a gene ontology term enrichment plot, run on genes found on the zebra finch GRC](https://CormacKinsella.github.io/assets/img/data_vis-GRC-GO-enrichment.png "Gene ontology enrichment on the zebra finch germline-restricted chromosome")

>Genes identified on the songbird germline-restricted chromosome (GRC) were subjected to GO term enrichment analysis, motivated by the unknown role of the GRC and its unusual genetic features.

<br/>

<center> <b> Tanglegram, from <a href="https://www.nature.com/articles/s41467-020-18474-w">Nature Communications, 2020</a> </b> </center>

![The image shows a tanglegram between two virus genes, uncovering recombination between two virus families that infect the same host](https://CormacKinsella.github.io/assets/img/data_vis-tanglegram.png "Recombination of genetic modules between two virus families (Naryaviridae & Nenyaviridae) infecting the same hosts")

>A tanglegram can reveal patterns of either concordant or discordant evolutionary history between two subjects. Here, connections link genes found on the same virus genome. The overall pattern reveals extensive and reciprocal recombination of genetic modules between two virus families (*Naryaviridae* & *Nenyaviridae*). Given these families had unknown hosts, we argued their ability to recombine supported a shared host, which we showed was *Entamoeba*.

<br/>

<center> <b>  Genome synteny, from <a href="https://www.pnas.org/doi/10.1073/pnas.2303844120">PNAS, 2023</a> </b> </center>

![Alt text](https://CormacKinsella.github.io/assets/img/data_vis-Draup-genome-synteny.png "text")

>Genome synteny plot, here examining whole avipoxvirus genomes to better understand some horizontally transferred genes from ssDNA viruses (marked in red). Despite substantial genome size disparity, overall gene order is highly conserved across the genus *Avipoxvirus*.

<br/>