---
layout: page
title: Visualisation of data
subtitle: Examples from prior studies
---

<br/>

<center> <b> Circos, from <a href="https://www.nature.com/articles/s41467-020-18474-w">Nature Communications, 2020</a> </b> </center>

![The image shows alignments between multiple contigs, visualised with the program Circos](https://CormacKinsella.github.io/assets/img/data_vis-treebeard-circos.png "Circos for visualising complex features and multiple alignments")

>Visualising complex features and multiple alignments. Here, showing sequence identity between virus genes and endogenous viral element regions in parasite genomes.

<br/>

<center> <b> Alphafold & Mol*, from <a href="https://www.pnas.org/doi/10.1073/pnas.2303844120">PNAS, 2023</a> </b> </center>

![The image shows protein structures, predicted by Alphafold, visualised with the program Mol*](https://CormacKinsella.github.io/assets/img/data_vis-PNAS_alphafold.png "Alphafold protein/domain structure analysis")

>Alphafold in combination with Mol* for protein domain analysis. Here, exploring predicted monomer and multimer structure of proteins involved with single-stranded DNA replication.

<br/>

<center> <b> Cell type identification in a 10X Genomics single-cell RNA-Seq dataset, unpublished </b> </center>

![The image shows a Seurat analysis of scRNA-Seq data. B cells are identifiable via expression of their specific markers](https://CormacKinsella.github.io/assets/img/data_vis-Seurat_B_cells.png "Single-cell RNA-Seq analysis with Seurat, identifying a B cell population")

>Single-cell RNA-Seq enables examination of cell type specific changes throughout disease states. Here, Seurat was used to cluster cells by their respective expression profiles. Expression of B cell markers was then used to identify and label that population of immune cells.

<br/>

<center> <b> Tanglegram, from <a href="https://www.nature.com/articles/s41467-020-18474-w">Nature Communications, 2020</a> </b> </center>

![The image shows a tanglegram between two virus genes, uncovering recombination between two virus families that infect the same host](https://CormacKinsella.github.io/assets/img/data_vis-tanglegram.png "Recombination of genetic modules between two virus families (Naryaviridae & Nenyaviridae) infecting the same hosts")

>A tanglegram can reveal patterns of either concordant or discordant evolutionary history between two subjects. Here, connections link genes found on the same virus genome. The overall pattern reveals extensive and reciprocal recombination of genetic modules between two virus families (*Naryaviridae* & *Nenyaviridae*). Given these families had unknown hosts, we argued their ability to recombine supported a shared host, which we showed was *Entamoeba*.

<br/>

<center> <b> Gene ontology enrichment, from <a href="https://www.nature.com/articles/s41467-019-13427-4">Nature Communications, 2019</a> </b> </center>

![The image shows a gene ontology term enrichment plot, run on genes found on the zebra finch GRC](https://CormacKinsella.github.io/assets/img/data_vis-GRC-GO-enrichment.png "Gene ontology enrichment on the zebra finch germline-restricted chromosome")

>Genes identified on the songbird germline-restricted chromosome (GRC) were subjected to GO term enrichment analysis, motivated by the unknown role of the GRC and its unusual genetic features.

<br/>

<center> <b> Genome synteny with Clinker, from <a href="https://www.pnas.org/doi/10.1073/pnas.2303844120">PNAS, 2023</a> </b> </center>

![The image shows a genome synteny plot between four individual species of the genus Avipoxvirus](https://CormacKinsella.github.io/assets/img/data_vis-Draup-genome-synteny.png "Genome synteny between avipoxviruses")

>Genome synteny plot, here examining whole avipoxvirus genomes to better understand some horizontally transferred genes from ssDNA viruses (marked in red). Despite substantial genome size disparity, overall gene order is highly conserved across the genus *Avipoxvirus*.

<br/>

<center> <b> Tissue-specific structural variant analysis with Loupe, from <a href="https://www.nature.com/articles/s41467-020-18474-w">Nature Communications, 2019</a> </b> </center>

![The image shows structural variation found in one tissue of an individual zebra finch, but not another](https://CormacKinsella.github.io/assets/img/data_vis-GRC-structural-variation.png "Structural variation uncovered by 10X Genomics linked-reads")

>10X Genomics linked-reads came from a now discontinued pseudo long-read library preparation method. Barcode enrichment signified reads derived from physically close DNA regions. Here, the same genomic region is examined in two samples. In the left panel we see a library prepared from zebra finch liver (somatic cells), and on the right a library prepared from testis (including germline cells). Reads that map to chromosomes 1 and 3 are physically close in the testis but not the liver, supporting the existence of a structural variant (with respect to the somatic reference assembly). In fact this pattern comes from the "hybrid chromosome" known as the germline-restricted chromosome, which includes regions derived from all autosomes.

<br/>

<center> <b> Recombination hotspot analysis, from <a href="https://academic.oup.com/ve/article/8/2/veac087/6702273">Virus Evolution, 2022</a> </b> </center>

![The image shows patterns of modular recombination within the family Redondoviridae](https://CormacKinsella.github.io/assets/img/data_vis-vir_evo_modular_recomb.png "Recombination hotspots across redondovirus genomes")

>Patterns of recombination are useful to know for a variety of questions. Here, RDP4 was used to understand where recombination is most commonly observed in redondovirus genomes. In the lower left is a matrix of breakpoint pairs, essentially revealing that blocks of recombination usually touch the start and end of genes (that is, they rarely occur within genes). On the top right, windows of phylogenetic compatibility across the genomes show the same picture. Lower phylogenetic compatibility is found between genes than within them.

<br/>

<center> <b> Protein clustering, from <a href="https://www.pnas.org/doi/10.1073/pnas.2303844120">PNAS, 2023</a> </b> </center>

![The image shows proteins clustered with the CLANS software package](https://CormacKinsella.github.io/assets/img/data_vis-protein_clusters.png "Protein clustering with CLANS")

>Rapid insight into protein sequence relationships can be gained using clustering tools, such as the CLANS package. Here, virus capsid proteins found in both host genomes (as endogenous elements) and exogenous viruses, were analysed together to visualise their approximate relationships.

<br/>